### ISSUE TITLE
bruh where this shit at lmao

### ISSUE THREAD

**Issue by catchnicc** [Zakaria-Codex/APEX_ESP_AIMBOT#3]
An error occurred in 'apex' while attempting to open 'kRpm.h'

**Comment by catchnicc** [comment]
Error	C1083	Cannot open include file: 'd3dx9math.h': No such file or directory	
**this too**

**Comment by sysysysy242** [comment]
kRmp code : [kRpm.zip](https://github.com/Zakaria-Codex/APEX_ESP_AIMBOT/files/6262798/kRpm.zip)

[Include.zip](https://github.com/Zakaria-Codex/APEX_ESP_AIMBOT/files/6262803/Include.zip)
and for d3dx9math.h extract the include.zip in folder, go to project setting, VC an string and in the includefield, and write the acces to the folder

**Comment by catchnicc** [comment]
> kRmp code : [kRpm.zip](https://github.com/Zakaria-Codex/APEX_ESP_AIMBOT/files/6262798/kRpm.zip)
> 
> [Include.zip](https://github.com/Zakaria-Codex/APEX_ESP_AIMBOT/files/6262803/Include.zip)
> and for d3dx9math.h extract the include.zip in folder, go to project setting, VC an string and in the includefield, and write the acces to the folder

yeah I have no idea what you just said to me lol. I don't know much about coding just the basic compiling, I'm not good at writing or know language very well at all.

**Comment by catchnicc** [comment]
> > kRmp code : [kRpm.zip](https://github.com/Zakaria-Codex/APEX_ESP_AIMBOT/files/6262798/kRpm.zip)
> > [Include.zip](https://github.com/Zakaria-Codex/APEX_ESP_AIMBOT/files/6262803/Include.zip)
> > and for d3dx9math.h extract the include.zip in folder, go to project setting, VC an string and in the includefield, and write the acces to the folder
> 
> yeah I have no idea what you just said to me lol. I don't know much about coding just the basic compiling, I'm not good at writing or know language very well at all.

lemme clarify, I obviously understood kRmp portion. 
but I'm not sure where to go regarding d3dx9math.h and the include zip file

**Comment by sysysysy242** [comment]
do you have discord ? if yes dm me : Sysy's#3891
